resources/less/virkailija-application.less

```
@@ -675,12 +675,18 @@ and (max-width: 1000px) {
 .application__attachment-review-info-row {
   display: flex;
   align-items: flex-start;
-  margin-bottom: 10px;
+}
+
+.application__attachment-review-deadline-row {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  margin-top: 20px;
 }
 
 .application__attachment-review-row-attachments {
   padding: 0px;
-  margin: 0px;
+  margin: 20px 0px 0px 0px;
```

```suggestion
  margin-top: 20px;
```

`px`-yksikkö lienee myös turha, kun arvo on 0.

---

Pelkkä margin-top ei valitettavasti riitä, koska pitää poistaa ul elementin alamarginaali.